app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::V1::PostsController < ApplicationController
+  def index
+    @posts = Post.where(author_id: user.id)
+    render json: @posts
+  end
+
+  def show
+    render json: @post
```

Comment:
**You both did great job!**

[OPTIONAL] It would be great idea if show page is working as expected. Currently it's returning `null` what you need to do is... accept my suggestion. 

```suggestion
    @post = Post.find(params[:id])
    render json: @post
```

_This is just for post. It would ge great if you take care of other controllers as well :). Thank you!_